functorch/_src/partitioners.py

```diff
@@ -346,6 +365,9 @@ def is_tensor_node(x):
         # Natalia said that we should allow recomputing indexing :)
         default_recomputable_ops += [aten.index]
 
+        # add more generally ?
+        default_recomputable_ops += pointwise_ops()
```

Comment:
I recalled the DTensor pointwise op set might not be a complete set of all the pointwise operators in ATen (although it covers most of them). Do you need to include all possible pointwise ops initially, or we can increase the coverage step by step?

Reply:
Does dtensor need the narrower set here?

Reply:
Yeah possibly DTensor needs a narrower set as some of the pointwise op might not make too much sense to DTensor, but I would prefer the tag in native_functions to reflect all pointwise ops, and leave the subsystems (i.e. DTensor or aot autograd) to decide and filter out the ops needed from the tagged set.